package/build-and-ship.sh

```diff
@@ -25,6 +25,6 @@ gcloud compute scp --tunnel-through-iap "$PKGFILE" "$FREIGHT_HOST:/tmp/$PKGFILE"
 # shellcheck disable=SC2087
 gcloud compute ssh --tunnel-through-iap "$FREIGHT_HOST" -- -T <<END
 sudo freight add "/tmp/${PKGFILE}" apt/bionic apt/focal
```

Comment:
This command should also run as the `freight` user. It doesn't cause problems directly but can interfere with cleaning up packages later.